tests/std/tests/P0943R6_stdatomic_h/test.compile.pass.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <stdatomic.h>
+
+static_assert(ATOMIC_BOOL_LOCK_FREE == 2);
+static_assert(ATOMIC_CHAR_LOCK_FREE == 2);
+static_assert(ATOMIC_CHAR16_T_LOCK_FREE == 2);
+static_assert(ATOMIC_CHAR32_T_LOCK_FREE == 2);
+static_assert(ATOMIC_WCHAR_T_LOCK_FREE == 2);
+static_assert(ATOMIC_SHORT_LOCK_FREE == 2);
+static_assert(ATOMIC_INT_LOCK_FREE == 2);
+static_assert(ATOMIC_LONG_LOCK_FREE == 2);
+static_assert(ATOMIC_LLONG_LOCK_FREE == 2);
+static_assert(ATOMIC_POINTER_LOCK_FREE == 2);
+
+#include <atomic>
+#include <type_traits>

```

Comment:
Usually we do `using namespace std;` in test code

Reply:
using `using namespace std;` here would defeat the whole purpose of this test. It's job is to make sure that all global names correspond to the names in `std` namespace.

Reply:
Makes sense! If you want, you could add a comment like the one in the UDL test (which also really cares about namespaces):

https://github.com/microsoft/STL/blob/1866b848f0175c3361a916680a4318e7f0cc5482/tests/std/tests/Dev11_0000000_user_defined_literals/test.cpp#L45

It would also be fine to leave the code as-is.